Resources/Audio/Voice/Cluwne/attributions.yml

```diff
@@ -0,0 +1,14 @@
+- files: ["cluwnelaugh1.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Created by Nanakisan and downloaded from FreeSound.org. Modified by github: brainfood1183"
+  source: "https://freesound.org/people/Nanakisan/sounds/253525/"
+
+- files: ["cluwnelaugh2.ogg"]
+  license: "CC-BY-4.0"
+  copyright: "Created by Nanakisan and downloaded from FreeSound.org. Modified by github: brainfood118"
```

Comment:
```suggestion
  copyright: "Created by Nanakisan and downloaded from FreeSound.org. Modified by github: brainfood1183"
```

Reply:
AJCM made a suggestion on how these should be presented, i will change them to fit hisa suggestion which would be. Made by Nanakisan (FreeSound.org), Modified by brainfood1183 (github) for ss14.